root/user/components/DownvotedSwitch.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../../context';
+import uriWith from '../../utility/uriWith';
+
+type Props = {
+  +$c: CatalystContextT,
+  +showDownvoted: boolean,
+};
+
+const DownvotedSwitch = ({
+  $c,
+  showDownvoted,
+}: Props) => (
+  <p>
+    {showDownvoted ? (
+      exp.l(
+        `Showing entities where the user voted against this tag.
+        {upvotes_link|Show their votes for the tag instead}.`,
+        {upvotes_link: uriWith($c.req.uri, {show_downvoted: 0})},
+      )
+    ) : (
+      exp.l(
+        `Showing entities where the user voted for this tag.
+        {downvotes_link|Show their votes against the tag instead}.`,
+        {downvotes_link: uriWith($c.req.uri, {show_downvoted: 1})},
+      )
+    )}
```

Comment:
I wonder if we should change the page headings from just `Tag “foo”` to `Votes for tag “foo”` and `Votes against tag “foo”` depending on what's being viewed, instead of explaining it down here. It'd then be nice to make the link text a bit more descriptive here by including the user/tag in the link text, e.g. `"Show {upvotes_link|entities {user} has tagged “{tag}”}.` and `"Show {downvotes_link|entities where {user} has voted against “{tag}”}.`. What do you think? :)

Reply:
Seems sensible. Done!